Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: clean package package-deps package-source package-upload package-wheel
+
+package-deps:
+	pip install --upgrade twine wheel
+
+package-source:
+	python setup.py sdist
+
+package-wheel: package-deps
+	python setup.py bdist_wheel --universal
```

Comment:
I'm not sure the `--universal` flag is correct here, since we only partially support Python 3. (See https://packaging.python.org/distributing/#universal-wheels) However, if we remove it, I suspect we'll forget to add it back once Python 3 support is more robust.  Thoughts?